File: backend2/DataAccess/DaoGenerated/システム共有/tt_afbatchlogDto.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: DTO定義
//             バッチログテーブル
// 作成日　　: 2024.07.23
// 作成者　　: 
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afbatchlogDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afbatchlog";
		public long batchlogseq { get; set; }                     //バッチログシーケンス
		public long sessionseq { get; set; }                      //セッションシーケンス
		public DateTime syoridttmf { get; set; }                  //処理日時（開始）
		public DateTime syoridttmt { get; set; }                  //処理日時（終了）
		public string? msg { get; set; }                          //メッセージ
		public string? pram { get; set; }                         //パラメータ
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afbatchlogDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             バッチログテーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tt_afbatchlogDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afbatchlog"
        Public Property batchlogseq As Long                     'バッチログシーケンス
        Public Property sessionseq As Long                      'セッションシーケンス
        Public Property syoridttmf As Date                  '処理日時（開始）
        Public Property syoridttmt As Date                  '処理日時（終了）
        Public Property msg As String                          'メッセージ
        Public Property pram As String                         'パラメータ
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
    End Class
End Namespace
